org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -28,6 +28,9 @@
   <li>Exclude from a report a part of bytecode that compiler generates for a
       synchronized statement
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/501">#501</a>).</li>
+  <li>Exclude methods generated by Lombok. Initial analysis and contribution by
```

Comment:
@marchof I'd prefer to be more explicit here and state that
1. they are excluded from report - to avoid confusion with exclusion from instrumentation
2. only methods annotated with `lombok.Generated` are excluded - because annotation can be turned off

For consistency with previous changelog entries - we don't put dot before opening parenthesis.

So something like:
```
Exclude from a report methods annotated by <code>lombok.Generated</code>,
initial analysis contributed by Rüdiger zu Dohna (GitHub ...).
```